lib/wibox/layout/ratio.lua

```diff
@@ -334,18 +334,20 @@ function ratio:ajust_ratio(index, before, itself, after)
 
     self:emit_signal("widget::layout_changed")
 end
+ratio.ajust_ratio = ratio.adjust_ratio -- For backwards compatibility TODO v5: Remove this
 
 --- Update all widgets to match a set of a ratio.
 --
--- @method ajust_widget_ratio
--- @tparam widget widget The widget to ajust
+-- @method adjust_widget_ratio
+-- @tparam widget widget The widget to adjust
 -- @tparam number before The sum of the ratio before the widget
 -- @tparam number itself The ratio for "widget"
 -- @tparam number after The sum of the ratio after the widget
-function ratio:ajust_widget_ratio(widget, before, itself, after)
+function ratio:adjust_widget_ratio(widget, before, itself, after)
     local index = self:index(widget)
-    self:ajust_ratio(index, before, itself, after)
+    self:adjust_ratio(index, before, itself, after)
 end
+ratio.ajust_widget_ratio = ratio.adjust_widget_ratio -- For backwards compatibility TODO v5: Remove this
```

Comment:
```suggestion
--- Update all widgets to match a set of a ratio.
--
-- This method is kept for backwards compatibility, please use `:adjust_widget_ratio` instead.
-- @see wibox.layout.ratio.adjust_widget_ratio
-- @deprecated wibox.layout.ratio.ajust_widget_ratio
-- @method ajust_widget_ratio
-- @tparam widget widget The widget to adjust
-- @tparam number before The sum of the ratio before the widget
-- @tparam number before The sum of the ratio before the widget
-- @tparam number itself The ratio for "widget"
-- @tparam number itself The ratio for "widget"
-- @tparam number after The sum of the ratio after the widget
-- @tparam number after The sum of the ratio after the widget
function ratio:ajust_widget_ratio(...)
    require('gears.debug').deprecate(
        "Use `:adjust_widget_ratio` rather than `:ajust_widget_ratio`",
        { deprecated_in = 5 }
    )
    return self:adjust_widget_ratio(...)
end
```